### ISSUE TITLE
Control y axis ticks so it also shows a round number and has a sensible spacing

### ISSUE THREAD

**Issue by 3ll3d00d** [3ll3d00d/beqdesigner#24]
i.e. similar to how REW organises the y axis

**Comment by 3ll3d00d** [comment]
I am not going to make it user defined. 

The solution to this issue is to come up with the equivalent of the existing autolocator but which only uses nice round numbers. In order to do this sensibly you also need to respect, and possibly control, the axis range. This will be a fiddly and tedious thing to get right so not something I intend to tackle when the existing behaviour seems ok to me. 

If I can come up with a simple solution then I will add it though.

**Comment by aron7awol** [comment]
Are you using AutoLocator now?

What about using MaxNLocator with something like nbins=20, integer=True, and maybe something like steps=[1, 6, 10] or steps=[1, 1.5, 5, 6, 10] would also work well for the intervals?  The only thing I thought made sense to be user defined was nbins, so that the max number of ticks could be changed.  AutoLocator seems to just be a default version of MaxNLocator so it wouldn't work drastically differently than it does now, just with more flexibility?  I'm not entirely sure how well those sets of steps would work, but really, I'm totally fine with how it works now, I just wanted a lot more ticks, which MaxNLocator seems to be the perfect simple solution for.  If the steps also help make the intervals full minutes, then that's a bonus, but I'm not sure it would support steps=[1, 5, 10, 15, 60] or any step values over 10.

**Comment by 3ll3d00d** [comment]
yes i'm using the default setup now

like I said before, tedious and fiddly thing to work through and test to check it does sensible things at all times (which it usually doesn't)

**Comment by aron7awol** [comment]
I agree that adjusting the steps complicates things a lot and would require a lot of testing with little relative benefit.

MaxNLocator can be used without defining steps, though, which would allow nbins to be changed to be able to increase the number of ticks but still have it behave reliably the same way as AutoLocator does now otherwise.  A much simpler change maybe worth considering?

**Comment by 3ll3d00d** [comment]
I will certainly consider this whenever I pick up this issue

**Comment by 3ll3d00d** [comment]
I haven't changed anything

I'm not sure why yours has half the number of ticks, what resolution area you at? and size of window?

**Comment by 3ll3d00d** [comment]
need to choose a min/max no of ticks for the spectrum chart

I have put min = 8, max = 24, steps = [1,3,6] for now

then 

a film that is <2hr long will get 5min intervals (because it will pick 300s as the step)
a clip that is 4-10mins long will get 30s intervals
a clip that is <4min long will get 10s intervals

and so on

this seems a good balance to me 

@aron7awol any preference? 

![image](https://user-images.githubusercontent.com/2487509/48301220-30f67480-e4e2-11e8-9700-a3b415eab82f.png)

**Comment by 3ll3d00d** [comment]
see https://github.com/3ll3d00d/beqdesigner/releases/tag/0.5.0-beta.5
